R/bias.R

```diff
@@ -89,22 +89,16 @@ bias_sample <- function(true_values, predictions) {
 #'
 #' \deqn{
 #' B_t = (1 - 2 \cdot \max \{i | q_{t,i} \in Q_t \land q_{t,i} \leq x_t\})
-#' \mathbf{1}( x_t \leq q_{t, 0.5}) \\
+#'  1( x_t \leq q_{t, 0.5}) \\
 #' + (1 - 2 \cdot \min \{i | q_{t,i} \in Q_t \land q_{t,i} \geq x_t\})
-#'  \mathbf{1}( x_t \geq q_{t, 0.5}),
-#' }{
-#' B_t = (1 - 2 * max(i | q_{t,i} in Q_t and q_{t,i} <= x_t\))
-#' 1( x_t <= q_{t, 0.5}) + (1 - 2 * min(i | q_{t,i} in Q_t and q_{t,i} >= x_t))
-#'  1( x_t >= q_{t, 0.5}),
-#' }
+#'  1( x_t \geq q_{t, 0.5}),}
 #'
 #' where \eqn{Q_t} is the set of quantiles that form the predictive
 #' distribution at time \eqn{t}. They represent our
-#' belief about what the true value \eqn{x_t} will be. For consistency, we
-#' define
+#' belief about what the true value $x_t$ will be. For consistency, we define
 #' \eqn{Q_t} such that it always includes the element
-#' \eqn{q_{t, 0} = - \infty} and \eqn{q_{t,1} = \infty}.
-#' \eqn{\mathbf{1}()}{1()} is the indicator function that is \eqn{1} if the
+#' \eqn{q_{t, 0} = - \infty and q_{t,1} = \infty}.
```

Comment:
Will this not remove the space, i.e. render to 
![image](https://github.com/epiforecasts/scoringutils/assets/1156307/bdc7b4fb-4679-4184-98d3-a6651e4a3ddc)

Reply:
I agree but this is just due to me having made bias_range use bias_quantile rather than its own implementation. They had disagreeing docs which you can see here. Will standardise on the "nicest" version before exiting draft.

Reply:
```suggestion
#' \eqn{q_{t, 0} = - \infty} and \eqn{q_{t,1} = \infty}.
```